Source/FlavoredNeutrinoContainerInit.cpp

```diff
@@ -494,6 +567,81 @@ InitParticles(const TestParams* parms)
 		  p.rdata(PIdx::Nbar) = ndensbar*scale_fac * (1. + 3.*parms->st4_fluxfacbar*costhetabar);
 		}
 
+		//====================//
+		// 5- Minerbo Closure //
+		//====================//
+		else if(parms->simulation_type==5){
+		  AMREX_ASSERT(NUM_FLAVORS==3 or NUM_FLAVORS==2);
+
+		  // set energy to 50 MeV
+		  p.rdata(PIdx::pupt) = 50. * 1e6*CGSUnitsConst::eV;
+		  p.rdata(PIdx::pupx) = u[0] * p.rdata(PIdx::pupt);
+		  p.rdata(PIdx::pupy) = u[1] * p.rdata(PIdx::pupt);
+		  p.rdata(PIdx::pupz) = u[2] * p.rdata(PIdx::pupt);
+		  
+		  // get the cosine of the angle between the direction and each flavor's flux vector
+		  Real mue = fluxfac_e>0 ? (parms->st5_fxnue*u[0] + parms->st5_fynue*u[1] + parms->st5_fznue*u[2])/fluxfac_e : 0;
+		  Real mua = fluxfac_a>0 ? (parms->st5_fxnua*u[0] + parms->st5_fynua*u[1] + parms->st5_fznua*u[2])/fluxfac_a : 0;
+		  Real mux = fluxfac_x>0 ? (parms->st5_fxnux*u[0] + parms->st5_fynux*u[1] + parms->st5_fznux*u[2])/fluxfac_x : 0;
```

Comment:
should this evaluation of the third flavor angle stuff only happen if NUM_FLAVORS=3? Or is that only relevant if you're doing something to the p.rdata() (like in lines 601-602)?

Reply:
Ah, good question. "three-flavor" means electron, muon, and tauon neutrinos, but also those three for antineutrinos. So really six different types. Normally neutrino transport simulates electron neutrinos ("nue"), electron antineutrinos ("nua"), and muon+tauon neutrinos+antineutrinos ("nux"). So they lump four of the types into "nux". Hence, we divide by 4 to extract the energy density from only one of the flavors. If we are simulating only 2 flavors, we ignore the tauon neutrinos. If we are simulating 3, they get added in, too.

Reply:
The reason regular transport does this is because the four lumped species do essentially exactly the same thing, so it is just computationally cheaper to simulate it once than simulate the same thing duplicated four times.